Animated.Text: update propTypes (deprecated in 0.62)

sviande opened the pull request.

sviande committed:
Animated.Text: update propTypes (deprecated in 0.62)

mwegener-com commented:
I created a patch file for this and [#285](https://github.com/n4kz/react-native-material-textfield/pull/285/files), so you don't loose your changes after npm install / ci: 
[Patchfile](https://gist.github.com/mwegener-com/75b475b17bf0ae7bf8b79923d3753a38)

6pm reviewed (commented):
well done

sviande committed:
AnimatedComponent: set useNativeDriver prop
(https://github.com/react-native-community/releases/blob/master/CHANGELOG.md#deprecated-1)

sviande committed:
Package: new version 0.17.0
